apps/dapp/src/components/Pages/Core/NewUI/Home.tsx

```diff
@@ -91,23 +91,23 @@ const Home = ({ tlc }: { tlc?: boolean }) => {
 
   useEffect(() => {
     const fetchMetrics = async () => {
-      const { data: treasuryData } = await fetchGenericSubgraph(
+      const { data: treasuryData } = await fetchGenericSubgraph<any>(
```

Comment:
now we could add type check here, e.g. check `TxnHistoryTable.tsx` subgraph queries